filters/ThirdParty/filter_251_FutaFilterPrivacy/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "filterId": 250,
+  "name": "FutaFilter URL Privacy Guardner List",
+  "description": "Now the tracker is proliferating and can be seen everywhere. This list designed to against URL tracks, protect users privacy.",
```

Comment:
Nowadays, trackers are proliferating and can be seen everywhere. This list is designed to protect against tracking URL parameters to protect users' privacy.